src/common/snippets/README.md

```diff
@@ -0,0 +1,9 @@
+# SnippetS
+
+* [SnippetS design guide](./docs/snippets_design_guide.md)
```

Comment:
Please update key persons. I think it can be CPU plugin developers

@a-sidorova Please take a look

Reply:
@a-sidorova ping